src/app/search/search-results/search-results.component.scss

```diff
@@ -14,6 +14,23 @@
  *    limitations under the License.
  */
 
+mat-tab-group.homeComponent {
+  border: 0.0625rem solid #cfd1d7;
+  border-bottom-right-radius: 1rem;
+  border-bottom-left-radius: 1rem;
+}
+
+::ng-deep .mat-tab-header-pagination {
+  display: none !important;
```

Comment:
See if you can get rid of the ng-deep and !important.

Reply:
ng-deep and !important was the better workaround that I can find in removing the '< >' on the side of the tab headers. I have attached a screenshot to better depict the issue. The issue only shows up when the user is resizing the browser window vertically. 
<img width="934" alt="Screen Shot 2020-01-22 at 1 12 04 PM" src="https://user-images.githubusercontent.com/42463687/72934750-0a081500-3d19-11ea-8752-8374de2f5bbd.png">

The other workaround would be to insert:
`document.getElementsByClassName('mat-tab-header-pagination-before')[0].remove();
  document.getElementsByClassName('mat-tab-header-pagination-after')[0].remove();`
into ngOnInit()

Apparently, a user in stackoverflow said that this is a known issue that exists in angular material tabs.
https://stackoverflow.com/questions/57888846/set-horizontal-scroll-in-angular-material-tabs

Reply:
Stick:
```
mat-tab-group.no-pagination {
  .mat-tab-header-pagination {
    display: none;
  }
}
```

Reply:
Done.